plan/plan.go

```diff
@@ -146,6 +146,12 @@ func (p *Plan) Calculate() *Plan {
 
 	for _, topRow := range t.rows {
 		for _, row := range topRow {
+			// We ignore defined Endpoints with unknown targets, as there seems to be
```

Comment:
suggest: Can we provide a real world use case that explains this scenario? Otherwise this new condition might change the way existing deployments work (probably leading to issues, incidents, etc.)

We might also be prompted to make this a feature flag if this is not a bug.